src/wallet/rpcwallet.cpp

```diff
@@ -3444,6 +3444,7 @@ static RPCHelpMan bumpfee_helper(std::string method_name)
                              "are replaceable).\n"},
                     {"estimate_mode", RPCArg::Type::STR, /* default */ "unset", std::string() + "The fee estimate mode, must be one of (case insensitive):\n"
     "         \"" + FeeModes("\"\n\"") + "\""},
+                    {"add_inputs", RPCArg::Type::BOOL, /* default */ "true", "For a transaction with existing inputs, automatically include more if they are not enough."},
```

Comment:
Default should probably be "as needed", and let "true" (perhaps not now, but someday) be used to force additional inputs?